private/scripts/helpers.sh

```diff
@@ -1,5 +1,46 @@
 #!/usr/bin/env bash
```

Comment:
optional: prefer bash strict mode.
```
#!/bin/bash
set -euo pipefail
IFS=$'\n\t'
```

Reply:
Can't exit on failures because some of those failures we need to see in the script (specifically the Sage install script) and exiting would break things that are otherwise handled gracefully.

Reply:
Imo anything with the need to be handled gracefully should be wrapped in an if statement accordingly but fine as is.